pkg/network/etw/etw-utils.go

```diff
@@ -261,7 +261,7 @@ func parseUnicodeString(data []byte, offset int) (val string, nextOffset int, va
 		// wasn't null terminated.  Assume it's still a valid string though
 		lenString = len(data) - offset
 	} else {
-		lenString = termZeroIdx - offset
+		lenString = termZeroIdx
```

Comment:
I changed b/c I was debugging and it was cutting off the string at the wrong place.
If you look at it `term0idx` is already zero based from offste (because the call to `bytesOfDoubleZero` starts at `offset:`.  So the old code ended up effectively subtracting `offset` twice.

Reply:
damn you are right, I did not notice it